tools/syz-testbed/testbed.go

```diff
@@ -82,150 +78,220 @@ func main() {
 		tool.Failf("failed to parse manager config: %s", err)
 	}
 	if managerCfg.HTTP == "" {
-		managerCfg.HTTP = ":50000"
+		managerCfg.HTTP = "0.0.0.0:0"
 	}
 
-	checkouts := []*CheckoutInfo{}
-	for _, co := range cfg.Checkouts {
-		checkouts = append(checkouts, newCheckout(co, cfg, managerCfg))
+	ctx := TestbedContext{
+		Config:        cfg,
+		ManagerConfig: managerCfg,
 	}
-
-	log.Printf("------------------")
-	for _, co := range checkouts {
-		for _, instance := range co.Instances {
-			go runInstance(instance)
+	if cfg.HTTP != "" {
```

Comment:
I think it's better to provide ":0" default and always start the http server.

Reply:
But what would be the use of a server which is bound to some random and unknown port?

Reply:
We would need to print the actual port to the console.